pkg/logging/logging.go

```diff
@@ -47,7 +47,11 @@ var (
 	// DefaultLogLevelStr is the string representation of DefaultLogLevel. It
 	// is used to allow for injection of the logging level via go's ldflags in
 	// unit tests, as only injection with strings via ldflags is allowed.
-	DefaultLogLevelStr = "info"
+	DefaultLogLevelStr string = "info"
+
+	// DefaultLogFormatStr is the string representation of the default logrus.Formatter
+	// we want to use (possible values: text or json)
+	DefaultLogFormatStr string = "text"
```

Comment:
I think it would make sense to have two option variables, `LogFormatText` and `LogFormatJSON`, then set this to `LogFormatText` and use these variables everywhere else in these changes where the direct text `"text"` or `"json"` is used. This helps developers to find all references to the same option, and also ensures we don't hit any typo bugs.

Reply:
absolutely! I reviewed the logic with this approach, let me know what you think!